lib/routes/upc/jwc.ts

```diff
@@ -58,33 +36,23 @@ const handler = async (ctx) => {
     const items = await Promise.all(
         list.map((item) =>
             cache.tryGet(item.link, async () => {
-                const { data: response } = await got(item.link);
-                const $ = load(response);
-                // 选择类名为“comment-body”的第一个元素
-                item.description = $('.read').first().html();
-                // item.pubDate = $('.arti_update').html() === null ? '' : $('.arti_update').html().slice(5, 15);
-                // item.publisher = $('.arti_publisher').html();
-                item.author = $('.arti_publisher').html();
-                // console.log($('.arti_update').html().slice(5, 15));
-                // 上面每个列表项的每个属性都在此重用，
-                // 并增加了一个新属性“description”
+                try {
+                    const { data: response } = await got(item.link);
+                    const $ = load(response);
+                    // 选择类名为“comment-body”的第一个元素
+                    item.description = $('.read').first().html() || '无法获取正文内容，请手动访问';
+                } catch {
+                    item.description = '正文内容获取失败';
+                    return item;
```

Comment:
Unnecessary return statement since you do have a return statement right after the try-catch block.